modules/artifact/src/main/java/org/mule/runtime/module/artifact/api/descriptor/BundleDependency.java

```diff
@@ -21,6 +22,7 @@
   private BundleDescriptor descriptor;
   private BundleScope scope;
   private URI bundleUri;
+  private Set<BundleDependency> additionalDependencies;
```

Comment:
It doesn't seem right for me to have a BundleDependency with additionalDependencies. 
It feels like it's not a proper Object model. Could't that information be in the ClassLoaderModel instead?